src/containers/flows/AcceptBid/components/utils.ts

```diff
@@ -0,0 +1,40 @@
+//- Lib Imports
+import { ethers } from 'ethers';
+
+//- Contants Imports
+import { CURRENCY } from 'constants/currency';
+
+// Truncated NFT domain
+export const truncatedDomain = (domainName: string) => {
+	let domainText;
+	if (('wilder.' + domainName).length > 45) {
+		domainText = `wilder...${
+			domainName.split('.')[domainName.split('.').length - 1]
+		}`;
+		return domainText;
+	} else {
+		domainText = `${domainName}`;
+		return domainText;
+	}
+};
+
+// Truncated wallet address
+export const truncatedAddress = (walletAddress: string) => {
+	const truncatedWalletAddress = `${walletAddress.substring(
+		0,
+		4,
+	)}...${walletAddress.substring(walletAddress.length - 4)}`;
+	return truncatedWalletAddress;
+};
```

Comment:
These ones are defined `lib/utils` - import them from there to reduce code duplication